src/SlashCommands.js

```diff
@@ -282,6 +282,63 @@ var commands = {
             }
         }
         return reject(this.getUsage());
+    }),
+
+    // Verify a user, device, and pubkey tuple
+    verify: new Command("verify", "<userId> <deviceId> <deviceSigningKey>", function(room_id, args) {
+        if (args) {
+            var matches = args.match(/^(\S+) +(\S+) +(\S+)?$/);
```

Comment:
Shouldn't the last capture group (for `deviceSigningKey`) be mandatory? Otherwise, line 299 can get an undefined. And then line 301 would be comparing the device fingerprint against the input key.